packages/combobox/test/benchmark/basic-test-lightDOM.ts

```diff
@@ -0,0 +1,280 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import '@spectrum-web-components/combobox/sp-combobox.js';
+import '@spectrum-web-components/menu/sp-menu-item.js';
+import { html } from '@spectrum-web-components/base';
+import { measureFixtureCreation } from '../../../../test/benchmark/helpers.js';
+
+const countryList = [
```

Comment:
Does the test still work if we move this list to a sibling `index.ts` file that is shared between the two tests?

Reply:
No, but it works if `test/index.ts`, rather than `test/benchmark/index.ts`. Putting it in the benchmark folder makes it think the file is another benchmark test.